packages/prosess-vilkar-opptjening/src/components/OpptjeningVilkarAksjonspunktPanel.jsx

```diff
@@ -97,7 +97,7 @@ OpptjeningVilkarAksjonspunktPanelImpl.defaultProps = {
 export const buildInitialValues = createSelector(
   [ownProps => ownProps.behandlingsresultat, ownProps => ownProps.aksjonspunkter, ownProps => ownProps.status],
   (behandlingsresultat, aksjonspunkter, status) => ({
-    ...VilkarResultPicker.buildInitialValues(behandlingsresultat, aksjonspunkter, status),
+    ...VilkarResultPicker.buildInitialValues(behandlingsresultat, aksjonspunkter, status), // TODO (Hallvard) bytte fra behandlingsresultat til riktig vilår
```

Comment:
Hva innebærer disse todo'ene? Er det noe som er feil i koden som ligger her nå?
Det er også en i ErOmsorgVilkaarOppfyltForm.jsx

Reply:
Strukturen på dataene fra backend er endret så må endre hva som sendes inn her. Får ikke opp dette panelet nå, så må ta hånd om dette når man begynner på pleiepenger. Vi har en egen variant av denne pakken for omsorgspenger, mulig vi kan bruke den begge steder dersom det skal være perioder på pleiepenger også.